Index: src/kolmykov_shishkin_stepanov/KruskalAlgorithm.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package kolmykov_shishkin_stepanov;\r\n\r\npublic class KruskalAlgorithm {\r\n    private int num = 0;\r\n    private Node[] nodes;\r\n\r\n    public void setNumOfNodes(int num) {\r\n        nodes = new Node[num];\r\n        for (int i = 0; i < num; i++) {\r\n            nodes[i] = new Node(i);\r\n            Window.getGraphicsPanel().drawNode(i, num);\r\n        }\r\n    }\r\n\r\n    public void addVertex(int number1, int number2, int value) {\r\n        //TODO реализовать\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/kolmykov_shishkin_stepanov/KruskalAlgorithm.java	(revision 44713e27c4b2c15ca36e88617ea04f4ac1d888d6)
+++ src/kolmykov_shishkin_stepanov/KruskalAlgorithm.java	(date 1593694791249)
@@ -12,7 +12,7 @@
         }
     }
 
-    public void addVertex(int number1, int number2, int value) {
+    public void addEdge(int number1, int number2, int value) {
         //TODO реализовать
     }
 }
